portfolio/src/main/webapp/script.js

```diff
@@ -72,14 +72,19 @@ function loadMap() {
         position: desiredLocationLatLng, 
         map: map,
         animation: google.maps.Animation.DROP,
-        title: 'Travel Destinations'
+        title: 'Travel Destinations: Click me to see more!'
       });
+    const infowindow = new google.maps.InfoWindow({
```

Comment:
```suggestion
    const infoWindow = new google.maps.InfoWindow({
```

To be consistent with camel case per style guide.